mobilenet/quantize.py

```
@@ -0,0 +1,203 @@
+            # layerwise equalization
+            quant_setting.equalization = True
+            quant_setting.equalization_setting.iterations = 4
+            quant_setting.equalization_setting.value_threshold = 0.4
+            quant_setting.equalization_setting.opt_level = 2
+            quant_setting.equalization_setting.interested_layers = None
+            # replace ReLU6 with ReLU
+            model = convert_relu6_to_relu(model)
+        else:
+            raise ValueError(
+                "Please set optim_quant_method correctly. Support 'MixedPrecision_quantization', 'LayerwiseEqualization_quantization'"
+            )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)